config/fund_loader_config/cof/cof_r2.py

```diff
@@ -63,8 +63,13 @@
         "all_uploaded_documents_section_available": True,
         "application_fields_download_available": False,
         "display_logo_on_pdf_exports": False,
-        "requires_feedback": False,
         "mark_as_complete_enabled": False,
+        "feedback_survey_config": {
+            "requires_survey": False,
+            "requires_section_feedback": False,
+            "isSurveyOptional": True,
+            "isSectionFeedbackOptional": True,
```

Comment:
Should these be camelCase? seems inconsistent with the rest which are snake_case

Reply:
Also, aren't `requires_survey` and `isSurveyOptional` just opposites? 

Should `requires_survey` maybe be `has_feedback_survey`

Same logic for section feedback.  Not sure if I'm misunderstanding though!

Reply:
`requires_survey` and `isSurveyOptional` are not opposites, I have set the default values to turn off the survey

Reply:
I'm treating individual section feedback and surveys separately here

Reply:
I'll fix those camelcase stuff

Reply:
👍 nice fix!